code/modules/holiday/holidays.dm

```diff
@@ -350,7 +350,7 @@
 
 /datum/holiday/halloween
 	name = HALLOWEEN
-	begin_day = 26
+	begin_day = 10 // yogs - Changed halloween start day
```

Comment:
no, change this to the 24th at the very earliest.

Reply:
it used to be the 10th when was it changed